Mathlib/Logic/Encodable/Basic.lean

```diff
@@ -665,6 +665,16 @@ theorem le_sequence (a : α) : f a ≤ f (hf.sequence f (encode a + 1)) :=
   hf.rel_sequence a
 #align directed.le_sequence Directed.le_sequence
 
+variable (hf : Directed (· ≥ ·) f)
```

Comment:
Can you please put sections around sequence_mono and le_sequence on one hand, and around sequence_anti and sequence_le on the other hand, in order to restrict the scope of the `variable` declarations?

By the way, thanks a lot for moving these theorems. I left those post-port tasks and should have done them a while ago.

Reply:
sections added.